### PR TITLE
CONTACT-FORM

### PR DESCRIPTION
**Pull request by citec-47**
**###ADDITION OF CONTACT-FORM ON MY MOBILE PORTFOLIO. ✔**
##Here Are the Requirements and Method Used in Implementing This Project.
#General requirements🧾
- Make sure that there are [no linter errors] (https://github.com/microverseinc/linters-config).
 -Make sure that you used correct [GitHub flow] (https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git--github/articles/github_flow.md).
-Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-- -skills/blob/main/documentation/articles/professional_repo_rules.md).
-HTML/CSS requirements
#Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html--css/blob/main/articles/html_css_best_practices.md).
Project requirements✔
The section you need to build for this milestone is:
the contact form:✔
#Form
 and the footer (if present in the design)
.
-Make sure that:
*Name field accepts up to 30 chars.
*Email field accepts only emails.
*Message field accepts up to 500 chars.
*All fields have placeholder text.
*All fields have the correct type.
*A user cannot submit the form if one or more of the fields is empty.
#Collect data by using the [Formspree service](https://formspree.io/html).
-In order to lay out the elements on the page you should use Flexbox.
-You don't need to implement any functionality that requires JavaScript.
-You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]--(https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
-Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+      <li>Codepen</li>
+      <li>GitLab</li>
+      <li>Terminal</li>
+    </ul>
+  </section>
+  <h1 class="line">
+    I'm always interested in hearing about new projects,
+     so if you'd like to chat please get in touch.
+   </h1>
+ <form action="https://formspree.io/f/meqwogjz" method="post">
```
Comment (vickymarz) [inline index.html:205]: - [ ] our contact section seems to have a horizontal scroll bar and spaces at the right of the page which is not a good user experience. Kindly fix it 👍🏽 

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @citec-47, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by citec-47** [comment]
##CHANGES DONE✔✔
#Adjustment of icon pictures✔
#Removal of the horizontal scroll✔

File: index.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+ </form>
+ <footer>
+  <ul class="intro">
+    <li><a href="#"><img alt="github-icon" src="./image/Vector3.png.png"></a></li>
+    <li><a href="#"><img alt="linkedin-icon" src="./image/Disabled.png"></a></li>
+    <li><a href="#"><img alt="media-icon" src="./image/Vector1.png"></a></li>
+    <li><a href="#"><img alt="media-icon" src="./image/Vector.png.png"></a></li>
+    <li><a href="#"><img alt="meduim-icon" src="./image/Vector2.png"></a></li>
+  </ul>
+ </footer>
```
Comment (Shedrack-Sunday) [inline index.html:233]: - [Optional ] Firstly, I must commend you for making the needed changes by the previous reviewer, However, there's a slight variation on your `button`(added border) and social links which do not tally with the Figma design. the idea behind making you stick to such designs is to build a skilled muscle in meeting future project requirements or clients' projects. I've attached a comparison view fro Figma and your design for your assessment. Good job 👏 

# Figma:
image

# Your Design:
image

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @citec-47       👋 
 
 # STATUS: APPROVED ✔️  🍾  🥳

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉

 
 
 ## Highlights
✔️  Proper use of Github features 💯 
✔️  well-crafted pull request 👍 
✔️ Linters are passing 👏 
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._
 
- [ ] I made a suggestion concerning the design of your `submit` button and footer social links, which I strongly suggest that you take into consideration. Please find a more detailed review of the comments below

 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**